Cargo.toml

```diff
@@ -65,7 +65,7 @@ nu-std = { path = "./crates/nu-std", version = "0.78.1" }
 nu-utils = { path = "./crates/nu-utils", version = "0.78.1" }
 
 nu-ansi-term = "0.47.0"
-reedline = { version = "0.18.0", features = ["bashisms", "sqlite"] }
```

Comment:
Can you revert the manual override outside the patch section? (see L163)

Reply:
It's done, would you please have a look again?  Not really sure if I'm doing right thing